### PR TITLE
Evidences object update

### PR DESCRIPTION
**Pull request by a-fishman**
Related Issue: https://github.com/ocsf/ocsf-schema/issues/1000

Description of changes:

Adding account, device, email, url, user to evidences in detection finding.

Signed-off-by: Anton Fishman anton.fishman@hunters.ai

### CODE REVIEW

File: objects/evidences.json [hunk 1]
```diff
@@ -51,6 +63,14 @@
     "src_endpoint": {
       "description": "Describes details about the source of the network activity that triggered the detection.",
       "requirement": "recommended"
+    },
+    "url": {
+      "description": "The URL object that pertains to the event or object associated to the activity that triggered the detection.",
+      "requirement": "recommended"
+    },
+    "user": {
```
Comment (floydtree) [inline objects/evidences.json:71]: Would be useful to clarify in the description when one should use `actor` vs. using `user`

**Review by floydtree — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ Thankyou! -->
 -->
 
 ## [Unreleased]
+* #### Added account, device, email, url, user to evidences in detection finding.
```
[the remark this answers is not in the document]
Comment (a-fishman) [reply CHANGELOG.md]: Fixed

File: objects/evidences.json [hunk 3]
```diff
@@ -51,6 +63,14 @@
...
+      "description": "The user that pertains to the event or object associated to the activity that triggered the detection.",
+      "requirement": "recommended"
     }
   },
   "constraints": {
```
[the remark this answers is not in the document]
Comment (a-fishman) [reply objects/evidences.json:72]: Fixed

File: objects/evidences.json (hunk 1 above)
Comment (a-fishman) [reply objects/evidences.json:67]: Added in user description that it's a target or somehow associated with the activity. 
So if actor - it's a source, user - target or some related user.

**Review by pagbabian-splunk — APPROVED** [review]

**Review by floydtree — APPROVED** [review]

**Review by mikeradka — APPROVED** [review]

**Pull request merged**